NorthstarDLL/util/printmaps.cpp

```diff
@@ -32,6 +33,15 @@ std::vector<MapVPKInfo> vMapList;
 
 void RefreshMapList()
 {
+	// don't update our map list often from this func, only refresh every 10 seconds so we avoid constantly reading fs
+	static double fLastRefresh = -999;
+
+	if (fLastRefresh + 10.0 < R2::g_pGlobals->m_flRealTime)
+		fLastRefresh = R2::g_pGlobals->m_flRealTime;
+	else
+		return;
```

Comment:
Consider making the check conditional based on an argument, or calling the function `MaybeRefreshMapList`.